pipeline/generate-angular-http.sh

```diff
@@ -2,21 +2,44 @@
 
 SCRIPT_PATH=$( cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )
 
-PROJECT_ROOT_PATH="$SCRIPT_PATH/../"
-DOCKER_OPENAPI_PATH="/local/doc/api/openapi.yaml"
+DOCKER_OPENAPI_PATH="/local/openapi.yaml"
 DOCKER_OUTPUT_PATH="/local/src/web/src/app/api/"
-PROJECT_GENERATED_PATH="$SCRIPT_PATH/../src/web/src/app/api"
+PROJECT_GENERATED_PATH="$SCRIPT_PATH/../src/web/src/app/"
+PROJECT_GENERATED_PATH_FOR_CLEANING="$SCRIPT_PATH/../src/web/src/app/api"
+
+SPLITTED_DIR_PATH="$SCRIPT_PATH/../doc/api"
+TMP_PATH="/tmp/svetit/"
+TMP_PATH_OPENAPI_PATHS="/tmp/svetit/paths"
+
+[ -d "$TMP_PATH" ] || mkdir -p "$TMP_PATH"
+
+cp -r ${SPLITTED_DIR_PATH}/* ${TMP_PATH}
+
+for file in "$TMP_PATH_OPENAPI_PATHS"/*; do
+    if [ -f "$file" ]; then
+      echo "$file"
+      sed -i '/XUserHeader.yaml/d' "$file"
+      sed -i '/XSessionHeader.yaml/d' "$file"
+      sed -i '/XSpaceIdHeader.yaml/d' "$file"
+      sed -i '/XSpaceRoleHeader.yaml/d' "$file"
+      sed -i '/  parameters:/ { N; /  parameters:\n  responses:/ { s/  parameters:\n//; } }' "$file"
+      sed -i 's/handler-//g' "$file"
```

Comment:
Давай так попробуем чтобы чуть быстрее это было:
```suggestion
      expressions=(
        '/XUserHeader.yaml/d' 
        ';/XSessionHeader.yaml/d'
        ';/XSpaceIdHeader.yaml/d'
        ';/XSpaceRoleHeader.yaml/d'
        ';s/handler-//g'
      )
      str=''
      for i in ${expressions[@]}; do str+=$i; done
      
      sed -i "$str" "$file"
      sed -i '/  parameters:/ { N; /  parameters:\n  responses:/ { s/  parameters:\n//; } }' "$file"
```